README.debugger

```diff
@@ -78,6 +78,17 @@ view will show 'pf' to indicate this.
   pf = segment:offset or linear address is paged out or
        not present according to page tables.
 
+Debugging a program and breaking on start
+-----------------------------------------
+
+Dosbox-x includes a built-in command to launch a program and
```

Comment:
The official name of the program is (as I understand) "DOSBox-X", so it would be good to match the capitalization.

Reply:
Good point. Fixed :)